src/bio/bio_buffer.c

```diff
@@ -1551,13 +1551,13 @@ bio_copy_run(struct bio_copy_desc *copy_desc, unsigned int copy_size,
 }
 
 int
-bio_copy_post(struct bio_copy_desc *copy_desc)
+bio_copy_post(struct bio_copy_desc *copy_desc, int err)
 {
 	int	rc;
 
-	rc = bio_iod_post(copy_desc->bcd_iod_src);
+	rc = bio_iod_post(copy_desc->bcd_iod_src, 0);
```

Comment:
same, not sure why don't pass rc to bio_iod_post()?

Reply:
same here.